xbmc/pvr/PVRManager.h

```diff
@@ -71,6 +71,12 @@ namespace PVR
     bool m_bStopped;
   };
 
+  enum ChannelSwitchMode
```

Comment:
If we don't need to convert the enum values to int we should use an enum class instead.